src/_sidebar.md

```diff
@@ -8,6 +8,7 @@
 
   - [Checkbox](docs/elements/checkbox.md)
   - [Datepicker](components/datepicker/readme.md)
+  - [Timepicker](components/timepicker/readme.md)
```

Comment:
The sidebar should be alphabetical. Move it a bit down

Reply:
Hover effects are missing. Eventually it does make sense to be able to select more times?

Reply:
On the bottom of the overlay there is "no data" displayed furthermore.